src/pages/play.astro

```diff
@@ -262,19 +132,19 @@ changeLanguage('en')
 
       & .actions {
         display: flex;
-        gap: 1rem;
+        gap: var(--space-xs);
       }
 
       & .amount {
-        margin-block-end: 0.5rem;
+        margin-block-end: var(--space-2xs);
       }
 
       & ul.events {
         font-family: var(--sl-font-system-mono);
         height: 20rem;
-        margin-block: 0.5rem;
-        padding-block-start: 0.5rem;
-        padding-inline-start: 0.5rem;
+        margin-block: var(--space-2xs);
```

Comment:
The text in the `ul` is significantly higher on a desktop screen size. I feel like we can make it a bit smaller. This is probably coming from the applied styles to `main`.

Reply:
i've tried to make it smaller, i think it kinda matches the current version now